ckanext/switzerland/plugin.py

```diff
@@ -125,6 +126,29 @@ def i18n_domain(self):
     def get_blueprint(self):
         return [ogdch_admin, ogdch_dataset]
 
+    # IFacets
+
+    def dataset_facets(self, facets_dict, package_type):
+        lang_code = sh.get_request_language()
+        facets_dict["keywords_" + lang_code] = toolkit._("Keywords")
+        del facets_dict["tags"]
+
+        return facets_dict
+
+    def group_facets(self, facets_dict, group_type, package_type):
+        lang_code = sh.get_request_language()
+        facets_dict["keywords_" + lang_code] = toolkit._("Keywords")
+        del facets_dict["tags"]
+
+        return facets_dict
+
+    def organization_facets(self, facets_dict, organization_type, package_type):
+        lang_code = sh.get_request_language()
+        facets_dict["keywords_" + lang_code] = toolkit._("Keywords")
+        del facets_dict["tags"]
+
```

Comment:
do you think it is worthwhile to have all those three functions call the same function to not have so much duplicated code? I think three lines is kind of on the verge, but I think it would actually make it more readable. What do you think?